docker/docker-compose.yml

```diff
@@ -22,7 +34,24 @@ services:
       timeout: 5s
       retries: 5
   postgres:
```

Comment:
Add line break

Reply:
added in https://github.com/ghostfolio/ghostfolio/pull/3614/commits/75a0cb301e886645838ebac32c9ebafa4c58b6a1